seqr/views/utils/export_utils.py

```diff
@@ -77,10 +81,11 @@ def export_multiple_files(files, zip_filename, file_format='csv', add_header_pre
                 content += '\n'.join([
                     DELIMITERS[file_format].join([row.get(key) or blank_value for key in header]) for row in rows
                 ])
-                if not isinstance(content, unicode):
-                    content = unicode(content, errors='ignore')
-                zip_file.writestr('{}.{}'.format(filename.encode('utf-8'), file_format), content)
+                if isinstance(content, str):
+                    content = content.encode('utf-8')
+                content = str(content, 'ascii', errors='ignore') # Strip unicode chars in the content
+                zip_file.writestr('{}.{}'.format(filename, file_format), content)
         temp_file.seek(0)
         response = HttpResponse(temp_file, content_type='application/zip')
-        response['Content-Disposition'] = 'attachment; filename="{}.zip"'.format(zip_filename.encode('utf-8'))
```

Comment:
why doesn't this need to be encoded anymore?

Reply:
Because after `zip_filename` is encoded, it will be casted back to `str` and become part of the result string.
Please refer to below experiments:
Python 3:
```
>>> zip_filename='n\xe2me'
>>> zip_filename
'nâme'
>>> z=zip_filename.encode('utf-8')
>>> z
b'n\xc3\xa2me'
>>> type(z)
<class 'bytes'>
>>> a='attachment; filename="{}.zip"'.format(zip_filename.encode('utf-8'))
>>> b='attachment; filename="{}.zip"'.format(zip_filename)
>>> type(a)
<class 'str'>
>>> type(b)
<class 'str'>
>>> a
'attachment; filename="b\'n\\xc3\\xa2me\'.zip"'
>>> b
'attachment; filename="nâme.zip"'
>>> a==b
False
```
If `zip_filename.encode('utf-8')` is called then the result `a` is actually not the one we want. It embeds a string `b'n\xc3\xa2me'` which is the binary encoding of `zip_filename`.

Python 2:
```
>>> zip_filename=u'n\xe2me'
>>> z=zip_filename.encode('utf-8')
>>> z
'n\xc3\xa2me'
>>> type(z)
<type 'str'>
>>> a=u'attachment; filename="{}.zip"'.format(zip_filename.encode('utf-8'))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1: ordinal not in range(128)
----- comment: error because casting `zip_filename.encode('utf-8')` to a `str` --------
>>> a='attachment; filename="{}.zip"'.format(zip_filename.encode('utf-8'))
>>> b='attachment; filename="{}.zip"'.format(zip_filename)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe2' in position 1: ordinal not in range(128)
----- comment: error because casting unicode `zip_filename` to a Python 2 `str` --------
>>> b=u'attachment; filename="{}.zip"'.format(zip_filename)
----- comment: this is our case because we import future 'unicode_literals' so that `'attachment; filename="{}.zip"'` is the same as `u'attachment; filename="{}.zip"'`. ---------
>>> type(a)
<type 'str'>
>>> type(b)
<type 'unicode'>
>>> print(a)
attachment; filename="nâme.zip"
>>> print(b)
attachment; filename="nâme.zip"
>>> a==b
__main__:1: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
False
```
Since `zip_filename` contains a non-ascii char, some of the castings will be failed because of no specification for 'errors' while encoding/decoding.

Reply:
My concern here is I don't think the actual HTTP header can support unicode, I think it was relying on that binary encoding. Can you please update the unit tests to use a filename that has unicode in it and make sure it works in both versions of python

Reply:
Sorry, I thought the `content-disposition` header could do encoding as the HTTP content. It actually does nothing. I found [an article](https://medium.com/@JeremyLaine/non-ascii-content-disposition-header-in-django-3a20acc05f0d) about this. It is really a little complicated. But I updated the code anyway. Please have a look to see if it is acceptable.

Reply:
There are some other functions that generate the `Content-Disposition` header. If the code for unicode work I will update all of them. And, it is better to validate the unicode filename from the front end.

Reply:
I tried it with `staff Pages/Anvil` and `Download AnVIL metadata for test_project_nâme`. According to the captured header below, the `content-disposition` header with Unicode looks working.
![image](https://user-images.githubusercontent.com/33550059/86266016-981edc00-bb92-11ea-98bd-f1bb94f35387.png)

Reply:
as I said, we should strip unicode out of the file name completely the same way we strip it out of the content

Reply:
I keep the existing behaviors for unicodes. It is not ideal because some browsers may not get the right file name. But, it doesn't get worse. And, it maybe is important to continue supporting unicodes in the scenarios which it supports today. Is that okay?

Reply:
I think it doesn't matter if the filename looks a bit weird so long as it can successfully download, so I think this is fine

Reply:
I'm quite confident that the latest code keeps the existing feature in Python 3 and won't break anything in Python 3. So I don't change it to trip out unicodes.

Reply:
Changed to stripping out all unicodes in the filename to remove the risk of unicodes in Http headers.